### PR TITLE
CI workflow

### LINKED ISSUE #21: Github actions for dependbot fail to push image
This is because dependbot has no access right to push docker-image which is i guess all right.

We should fix it in a way that if the pull request is from dependbot, we should not try to push image.

image

A/C:
- skip publish docker image workflow (like [here](https://github.com/kbss-cvut/record-manager-ui/blob/64ff5e3c96d988a20e8effcd585ff9958777122f/.github/workflows/publish-docker-image.yml#L17))
- add CI workflow build directly with npm (and execute tests if there are any) (like [here](https://github.com/kbss-cvut/record-manager-ui/blob/64ff5e3c96d988a20e8effcd585ff9958777122f/.github/workflows/merge-to-protected.yml))

### PR DESCRIPTION
**Pull request by palagdan**
Resolve #21

### CODE REVIEW

File: .github/workflows/merge-to-protected.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
+      - name: Install
+        run: npm install
+      - name: Build
+        run: npm run-script build
```
Comment (blcham) [inline .github/workflows/merge-to-protected.yml:22]: OK, although not sure if after migration to vite it still holds but we will see I guess after merge

**Review by blcham — CHANGES_REQUESTED** [review]
See my suggestions.

File: .github/workflows/merge-to-protected.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
```
[the remark this answers is not in the document]
Comment (palagdan) [reply .github/workflows/merge-to-protected.yml]: done

File: .github/workflows/merge-to-protected.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+    branches: [main]
```
[the remark this answers is not in the document]
Comment (palagdan) [reply .github/workflows/merge-to-protected.yml]: done

**Pull request merged**